Gruntfile.js

```diff
@@ -187,7 +187,8 @@ module.exports = function(grunt) {
                 resolve: {
                     modulesDirectories: [
                         'src/js/',
-                        'src'
+                        'src',
+                        'node_modules'
```

Comment:
Let's update the simple-style-loader package, then require the new version in this project. If you'd like to comment this out and add a note on working with a local package go for it.